packages/cli/test/setup-test-folder.ts

```diff
@@ -10,6 +10,7 @@ mkdirSync(baseDir, { recursive: true });
 const testDir = mkdtempSync(baseDir);
 mkdirSync(join(testDir, '.n8n'));
 process.env.N8N_USER_FOLDER = testDir;
+process.env.N8N_ENFORCE_SETTINGS_FILE_PERMISSIONS = 'false';
```

Comment:
Can we use the logger for these warnings in `InstanceSettings` instead of `console.warn` so that this line below takes effect? Or this didn't work?

https://github.com/n8n-io/n8n/blob/274fcf45d393d8db1d2fb5ae1e774a4c9198a178/packages/cli/src/config/index.ts#L20

Reply:
+1 for switching to the logger.
I'm working on moving Logger to `core`. Then we can remove all `console.*` code from `core`, and mock the logger in all these tests.

Reply:
@ivov Would love to, but `Logger` is defined in `cli` and `InstanceSettings` is in `core`. Maybe we should move `Logger` into `core` 🤔

Reply:
Got it, let's go with this meanwhile then.